bundles/org.eclipse.mita.program.ui/src/org/eclipse/mita/program/ui/contentassist/ProposalPriorityHelper.xtend

```diff
@@ -13,26 +13,24 @@
 
 package org.eclipse.mita.program.ui.contentassist
 
-import org.eclipse.mita.platform.PlatformPackage
-import org.eclipse.mita.program.ProgramPackage
 import org.eclipse.core.runtime.NullProgressMonitor
 import org.eclipse.emf.ecore.EObject
 import org.eclipse.jface.text.contentassist.ICompletionProposal
+import org.eclipse.mita.base.types.TypesPackage
+import org.eclipse.mita.platform.PlatformPackage
+import org.eclipse.mita.program.ProgramPackage
 import org.eclipse.xtext.ui.editor.contentassist.ConfigurableCompletionProposal
 import org.eclipse.xtext.ui.editor.contentassist.ContentProposalPriorities
 import org.eclipse.xtext.ui.editor.hover.html.XtextBrowserInformationControlInput
 import org.eclipse.mita.base.types.Event
-import org.eclipse.mita.base.types.TypesPackage
 
 class ProposalPriorityHelper extends ContentProposalPriorities {
 	
-	public static final String ADDITIONAL_DATA__CANDIDATE = 'candidate';
-	
+	protected static final int PRIORITY_PARAMETER = 670;
```

Comment:
Why did this get separated from the others?

Reply:
I just ordered it based on priority.